pkg/console/operator/operator.go

```diff
@@ -138,15 +135,23 @@ func (c *consoleOperator) Sync(obj metav1.Object) error {
 	logrus.Infof("started syncing operator %q (%v)", obj.GetName(), startTime)
 	defer logrus.Infof("finished syncing operator %q (%v) \n\n", obj.GetName(), time.Since(startTime))
 
-	operatorConfig := obj.(*consolev1.Console)
-	if err := c.handleSync(operatorConfig); err != nil {
+	operatorConfig := obj.(*operatorsv1.Console)
+
+	consoleConfig, err := c.consoleConfigClient.Get(api.ResourceName, metav1.GetOptions{})
+	if errors.IsNotFound(err) && CreateDefaultConsoleFlag {
+		logrus.Errorf("No console config found. Creating.")
+		c.consoleConfigClient.Create(c.defaultConsoleConfig())
```

Comment:
We need to at least log the error if the create fails. Otherwise we will have no way to debug failures creating the console config.